packages/utils/src/collate.ts

```diff
@@ -0,0 +1,26 @@
+import type { DistributiveOmit } from '@noaignite/types'
+
+export type CollateOptions = DistributiveOmit<Intl.CollatorOptions, 'usage'>
+
+/**
+ * Sorts a list of strings alphabetically using the given locales and options.
+ *
+ * @param list - The list of `string` values to sort.
+ * @param locales - The locales to use for sorting.
+ * @param options - {@link CollateOptions} configurable options.
+ *
+ * @returns A new list of strings sorted alphabetically.
```

Comment:
I think we should use `toSorted` to ensure that this statement is true.
From what i've heard, `sort` will modify the current instance of `list`.

Reply:
Nice catch. Geez, what a fail on my part!